pkg/virt-handler/spice-proxy/spice-proxy.go

```diff
@@ -0,0 +1,224 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+package spice_proxy
+
+//go:generate mockgen -source $GOFILE -package=$GOPACKAGE -destination=generated_mock_$GOFILE
+
+/*
+ ATTENTION: Rerun code generators when interface signatures are modified.
+*/
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"sync"
+
+	v1 "kubevirt.io/client-go/api/v1"
```

Comment:
maybe missing the logic behind this. why do we have our own client-go? why is it empty https://github.com/kubevirt/client-go

Reply:
@phoracek we want to make our client-go consumable by 3rd party projects without the need to use the import the complete kubevirt repo.

For that we
- moved the client into the `/staging` dir
- created a symlink from the `/vendor` dir to the `/staging` dir (that's what's being used here)
- will sync `staging` into the empty repo you found, not done yet